wagtail/wagtailsnippets/templates/wagtailsnippets/snippets/confirm_delete.html

```diff
@@ -1,13 +1,32 @@
 {% extends "wagtailadmin/base.html" %}
 {% load i18n %}
-{% block titletag %}{% blocktrans with snippet_type_name=model_opts.verbose_name %}Delete {{ snippet_type_name }} - {{ instance }}{% endblocktrans %}{% endblock %}
+{% block titletag %}
+    {% if count == 1 %}
+      {% blocktrans with snippet_type_name=model_opts.verbose_name %}Delete {{ snippet_type_name }} - {{ instances|first }}{% endblocktrans %}
+    {% else %}
+      {% blocktrans with snippet_type_name=model_opts.verbose_name_plural|capfirst %}{{ count }} {{ snippet_type_name }}{% endblocktrans %}
```

Comment:
We should leave out the `{% blocktrans %}` here, as there's nothing that can be usefully translated. (I _think_ `verbose_name_plural` will be translated automatically if it's been defined with `_(...)` in the model definition, but will double-check that)